ui-core/src/main/java/com/adyen/checkout/components/ui/PaymentInProgressView.kt

```diff
@@ -36,7 +36,7 @@ class PaymentInProgressView @JvmOverloads constructor(
     private val binding = ViewPaymentInProgressBinding.inflate(LayoutInflater.from(context), this)
 
     override fun initView(delegate: ComponentDelegate, coroutineScope: CoroutineScope, localizedContext: Context) {
-        if (delegate !is ActionDelegate) throw IllegalStateException("Unsupported delegate type")
+        if (delegate !is ActionDelegate) return
```

Comment:
Can we have a scenario where a non `ActionDelegate`s is passed here?

Reply:
Whoops I changed it to return to test something. Only `ActionDelegate`s should be passed, so I think throwing an exception is better